rails4-deps/2.3.5/Gemfile

```diff
@@ -6,16 +6,16 @@ gem 'airbrake', '4.3.0'
 gem 'axlsx', '~> 1.3.4'
 gem 'dalli', '~> 2.7', '>= 2.7.6'
 gem 'diplomat'
-gem 'hashie', '2.1.2'
+gem 'hashie', '3.5.7'
```

Comment:
A brief scan of https://github.com/intridea/hashie/blob/master/UPGRADING.md suggest that there could be breaking changes in the upgrade from 2.1 to newer versions. For example there are a dozen or so usages of `to_hash` that might be affected by the upgrade.

@saraid Since you've had more experience with this gem, does anything stand out as risky to you with this upgrade?